packages/repository/src/model.ts

```diff
@@ -262,6 +272,17 @@ export abstract class Entity extends Model implements Persistable {
     const idName = this.definition.idName();
     return entityOrData[idName];
   }
+  /**
+   * Get the identity value of the primary key(s) for a given entity
+   * instance or entity data object.
+   *
+   * @param entityOrData The data object for which to determine the primary
+   * key(s).
+   */
+  static getPrimaryKey(entityOrData: AnyObject): any {
```

Comment:
Since this function can return 1 or more keys I think `getPrimaryKeys` is more clear. Also instead of `any` shouldn't this be a `string[]`

Reply:
+1

I am little bit confused about the purpose of this function. Are you trying to get primary key names? We already have `idProperties()` for that, plus I don't see why that requires `entityOrData` argument.